webapp/src/main/java/com/box/l10n/mojito/service/tm/TMTextUnitVariantRepository.java

```diff
@@ -153,13 +154,15 @@ Page<TMTextUnitVariant> findAllVariantsForPushRuns(
                     "                   and previous_tuv.locale_id = l.id " +
                     "                   and previous_tuv.text_unit_id = base_tu.id " +
                     "where " +
-                    "   latest_tuv.included_in_localized_file = true and " + // Exclude rejected translations
-                    "   (previous_tuv.id is null " +
-                    "   or (latest_tuv.id != previous_tuv.id and latest_tuv.content_md5 != previous_tuv.content_md5)) "
+                    "   latest_tuv.included_in_localized_file = true " + // Exclude rejected translations
+                    "   and base_tucv.created_date >= :translationsFromDate " +
+                    "   and (previous_tuv.id is null " +
+                    "       or (latest_tuv.id != previous_tuv.id and latest_tuv.content_md5 != previous_tuv.content_md5)) "
     )
     List<TextUnitVariantDelta> findDeltasForRuns(
             @Param("repositoryId") Long repositoryId,
             @Param("localeIds") List<Long> localeIds,
             @Param("pushRunIds") List<Long> pushRunIds,
-            @Param("pullRunIds") List<Long> pullRunIds);
+            @Param("pullRunIds") List<Long> pullRunIds,
+            @Param("translationsFromDate") Date translationsFromDate);
```

Comment:
this is worth adding a small comment Date vs DateTime :D

Reply:
Added! :)